test/CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ file(GLOB SSC_TESTS ssc_test/*.cpp)
 file(GLOB SHARED_TESTS shared_test/*.cpp)
 file(GLOB TCS_TESTS tcs_test/*.cpp)
 file(GLOB INPUTS_SRC input_cases/*.cpp)
-set(SSC_SRC ../ssc/vartab.cpp ../ssc/core.cpp ../ssc/common.cpp)
+set(SSC_SRC ../ssc/core.cpp ../ssc/common.cpp ../ssc/vartab.cpp)
 # add files which need to be compiled with Test in order to be tested on Windows
```

Comment:
I really do not have many tests comments since they do not run currently. We can revisit when the tests are running on all platforms...

Reply:
please try the tests again